src/server/src/main/java/io/cassandrareaper/jmx/ClusterFacade.java

```diff
@@ -440,9 +440,7 @@ public String getLocalEndpoint(Node node) throws ReaperException, InterruptedExc
    */
   public boolean nodeIsAccessibleThroughJmx(String nodeDc, String node) {
     return DatacenterAvailability.ALL == context.config.getDatacenterAvailability()
-        || DatacenterAvailability.LOCAL == context.config.getDatacenterAvailability()
-        || (DatacenterAvailability.EACH == context.config.getDatacenterAvailability()
-            && context.jmxConnectionFactory.getAccessibleDatacenters().contains(nodeDc));
+        || context.jmxConnectionFactory.getAccessibleDatacenters().contains(nodeDc);
```

Comment:
local mode is not allowed to make cross-dc jmx connections either. only ALL can do that.

Reply:
making this change caused the assertion on the following line (in multiple-reaper integration tests) to fail…
https://github.com/thelastpickle/cassandra-reaper/blob/master/src/server/src/test/java/io/cassandrareaper/acceptance/BasicSteps.java#L1521